File: Modul2/Bahasa_C/PRAK204-2310817210012-Nazmi_Hakim.c
#include <stdio.h>
int main() {
    double jari_jarii, tinggii;
    scanf("%lf %lf", &jari_jarii, &tinggii);
    double volumez = 22.0 / 7.0 * jari_jarii * jari_jarii * tinggii;
    double luas_permukaanz = 2.0 * 22.0 / 7.0 * jari_jarii * (jari_jarii + tinggii);
    double kelilingz = 2.0 * 22.0 / 7.0 * jari_jarii;
    printf("Volume = %.2lf\n", volumez);
    printf("Luas = %.2lf\n", luas_permukaanz);
    printf("Keliling = %.2lf\n", kelilingz);
    return 0;
}

File: Modul2/Bahasa_Python/PRAK204-2310817210012-Nazmi_Hakim.py
input_str = input()
jari_jarii, tinggii = map(float, input_str.split())
volumez = 22/7 * jari_jarii ** 2 * tinggii
luas_permukaanz = 2 * 22/7 * jari_jarii * (jari_jarii + tinggii)
kelilingz = 2 * 22/7 * jari_jarii
print(f"Volume = {volumez:.2f}")
print(f"Luas = {luas_permukaanz:.2f}")
print(f"Keliling = {kelilingz:.2f}")
